lib/record_loader/application_record_loader.rb

```diff
@@ -9,5 +10,12 @@ class ApplicationRecordLoader < RecordLoader::Base
     # Uses the standard RailsAdapter
     # @see https://rubydoc.info/github/sanger/record_loader/RecordLoader/Adapter
     adapter RecordLoader::Adapter::Rails.new
+
+    def wip_list
+      wip_files = []
+      wip_files_path = Rails.root.join('config/default_records')
+      Find.find(wip_files_path) { |path| wip_files << path if /\wip\.yml$/.match?(path) }
+      wip_files
+    end
```

Comment:
This method is supposed to return names but it is returning file paths. The names are intended to list the **features**. For example, it should have an item "011_scrna_core_cdna_prep_submission_templates" instead of a path like "/Users/ay6/webapps/sequencescape/config/default_records/submission_templates/011_scrna_core_cdna_prep_submission_templates.wip.yml"

Ref: https://github.com/sanger/record_loader

Reply:
Thank you! Returning file names works!

Reply:
It should also return all or none depending on the config option. Where is it done?

Reply:
in dev env, in config/environments/development.rb, I added a line 'config.deploy_wip_pipelines = true' . But i tested, it didn't work. Whatever it set to true or false, all wip files are loaded. any suggestions?

in training/uat/prod, the flag is set in deployment project. Please review the codes if that is the right place.

Reply:
You need to check the config option and return the WIPs if it is true. You need to code that in. The option is new, it has no idea about how to use it. I suppose you will change wip_list method. Have a search in the SS code and see how config options are accessed. Return an empty array from the method or all WIPs based on the option.